doc/source/programs/gdal_vector_check_geometry.rst

```diff
@@ -32,6 +32,12 @@ The following checks are performed, depending on the input geometry type:
 
 Validity/simplicity checking is performed by the GEOS library and should be consistent with results of software such as PostGIS, QGIS, and shapely that also use that library. GEOS does not consider repeated points to be a cause of invalidity or non-simplicity.
 
```

Comment:
Unfortunately I can't slip OpenJUMP into the list because it is using the mother JTS...
Nice trick how you generate SVG from WKT.